include/updater/updater_config.hpp

```diff
@@ -0,0 +1,81 @@
+/*
+
+Copyright (c) 2016, Project OSRM contributors
```

Comment:
I think we removed it everywhere except for public headers we install. But even then it's questionable..


Also it's 2017 already, who wants to s/2016/2017/g ? :D